examples/geojson/geojson_export_test.nlogo

```diff
@@ -0,0 +1,739 @@
+extensions [gis]
+globals [dataset reimport]
+
+to-report verticies-to-tuples [list-of-verts]
+  report map gis:location-of list-of-verts
+end
+
+to test-metro
+  clear-all
+  set dataset gis:load-dataset "test_DC_metro_stations.geojson"
+  gis:set-world-envelope gis:envelope-of dataset
+  gis:set-drawing-color red
+  gis:draw dataset 1
+
+  gis:store-dataset dataset "out.geojson"
+  set reimport gis:load-dataset "out.geojson"
+
+  let originalPoints []
+  foreach gis:feature-list-of dataset [ vf ->
+    set originalPoints lput (map verticies-to-tuples (gis:vertex-lists-of vf)) originalPoints
+  ]
+  let reimportPoints []
+  foreach gis:feature-list-of reimport [ vf ->
+    set reimportPoints lput (map verticies-to-tuples (gis:vertex-lists-of vf)) reimportPoints
+  ]
+
+  if originalPoints != reimportPoints [
+    error "Geometries in Point don't match up"
+  ]
+
+
+end
+
+to test-Point
+	clear-all
+	set dataset gis:load-dataset "Point.geojson"
+	gis:set-world-envelope (list -180 180 -90 90)
+	gis:set-drawing-color red
+	gis:draw dataset 1
```

Comment:
I know it's just a test file, but we do try to use 2 spaces in our NetLogo models for indentation (no tabs).  I see this in a few of the other procedures in this file, too.